mne/viz/utils.py

```diff
@@ -1162,6 +1162,11 @@ def _compute_scalings(scalings, inst, remove_dc=False, duration=10):
     if not isinstance(inst, (BaseRaw, BaseEpochs)):
         raise ValueError('Must supply either Raw or Epochs')
 
+    # If there are no "auto" scalings, we can return early!
+    if all([scalings[ch_type] != 'auto'
+            for ch_type in set(inst.get_channel_types())]):
```

Comment:
```suggestion
            for ch_type in inst.get_channel_types(unique=True)]):
```

Reply:
Nice, I didn't know about that parameter 👍